src/iuse.h

```diff
@@ -225,13 +224,9 @@ class iuse
     // Helper for handling pesky wannabe-artists
     static int handle_ground_graffiti( player *p, item *it, const std::string prefix );
 
-    static void reset_bullet_pulling();
-    static void load_bullet_pulling(JsonObject &jo);
 protected:
     typedef std::pair<std::string, int> result_t;
     typedef std::vector<result_t> result_list_t;
```

Comment:
Are the two typedefs her still used anywhere? If not, they should be removed as well.